deploy/helm/aiq-aira/values.yaml

```diff
@@ -0,0 +1,101 @@
+# ------------------------------------------------------------
+# The following values are for the AIQ AIRA backend service.
+# ------------------------------------------------------------
+
+replicaCount: 1
+
+imagePullSecret:
+  name: "ngc-secret"
+  registry: "nvcr.io"
+  username: "$oauthtoken"
+  password: ""
+  create: true
+
+ngcApiSecret:
+  name: "ngc-api"
+  password: ""
+  create: true
+
+tavilyApiSecret:
+  name: "tavily-secret"
+  create: true
+  password: ""
+
+# The image repository and tag for the AIQ AIRA backend service.
+image:
+  baserepo: nvcr.io
+  repository: nvcr.io/nvstaging/blueprint/aira-backend
+  tag: v1.2.0-public-rc1
+  pullPolicy: Always
+
+# The service type and port for the main AIQ AIRA backend service
+service:
+  port: 3838
+
+backendEnvVars:
+  INSTRUCT_MODEL_NAME: "meta/llama-3.1-8b-instruct"
+  INSTRUCT_MODEL_TEMP: "0.0"
+  NEMOTRON_MAX_TOKENS: "5000"
+  INSTRUCT_MAX_TOKENS: "5000"
+  INSTRUCT_BASE_URL: "instruct-llm:8000"
+  INSTRUCT_API_KEY: "not-needed"
+  NEMOTRON_MODEL_NAME: "nvidia/llama-3_3-nemotron-super-49b-v1_5"
+  NEMOTRON_MODEL_TEMP: "0.5"
+  NEMOTRON_BASE_URL: "nim-llm:8000"
+  AIRA_APPLY_GUARDRAIL: "false"
+  RAG_SERVER_URL: ""
+  RAG_INGEST_URL: ""
+  
+nim-llm:
+  enabled: true
+  service:
+    name: "instruct-llm"
+  image:
+      repository: nvcr.io/nim/meta/llama-3.3-70b-instruct
+      pullPolicy: IfNotPresent
+      tag: "1.12.0"
+  resources:
+    limits:
+      nvidia.com/gpu: 1
+    requests:
+      nvidia.com/gpu: 1
+  model:
+    ngcAPIKey: ""
+    name: "meta/llama-3.3-70b-instruct"
+
+# -----------------------------------------------------------
+# The following values are for the AIQ AIRA frontend service.
+# ------------------------------------------------------------
+
+# The frontend application is a React web app. We recommend a NodePort so the frontend will be accessible at <your-node-ip>:3001
+frontend:
+  enabled: true
+  service:
+    type: NodePort
+    port: 3000
+    targetPort: 3000
+    nodePort: 30080
+  image:
+    repository: nvcr.io/nvstaging/blueprint/aira-frontend
```

Comment:
same as above, Will have to update this path once we release the public container